changes/views.py

```diff
@@ -237,3 +240,20 @@ def post(self, request, *args, **kwargs):
             )
 
         return Response(status=status.HTTP_202_ACCEPTED)
+
+
+class ReceiveWhatsAppSystemEvent(ReceiveWhatsAppBase):
+    serializer_class = ReceiveWhatsAppSystemEventSerializer
+
+    def post(self, request, *args, **kwargs):
+        self.validate_signature(request)
+        serializer = self.get_serializer(data=request.data)
+        if not serializer.is_valid():
+            # If this isn't an event that we care about
+            return Response(status=status.HTTP_204_NO_CONTENT)
```

Comment:
Shouldn't this rather be a 400 error? Since we're accepting all event types, if the serializer fails, it means that the request is in an invalid format, not that this isn't an event that we care about.